uk_bin_collection/tests/input.json

```
@@ -1824,13 +1824,13 @@
         "LAD24CD": "E08000022"
     },
     "NorthWestLeicestershire": {
+        "postcode": "DE74 2FZ",
```

Biome (2.4.15)

[error] The key postcode was already declared.

(lint/suspicious/noDuplicateObjectKeys)